src/Agent.Worker/Build/BuildCommandExtension.cs

```diff
@@ -16,6 +16,8 @@ public sealed class BuildCommandExtension : AgentService, IWorkerCommandExtensio
 
         public string CommandArea => "build";
 
+        public string SupportedHostType => string.Empty;
```

Comment:
Also, what is Empty means? no supported type?
What about use Enum, so we can have build/releases/test/all etc.